source/docs/hardware-reference/pigeon2/index.rst

```diff
@@ -31,25 +31,174 @@ Status Light Reference
+            margin: 5px;
+            background-color: darkgray;
+        }
+        .ledGroup {
+            display: inline-block;
+            height: 20px;
+        }
+        table.center, table.center th, table.center td {
+            border: 1px solid white;
+            border-collapse: collapse;
+            padding: 5px;
+            text-align: center;
+        }
+
+        .tableOverflow {
+            overflow: scroll;
+        }
+
+        td.overflow {
+            max-width: 550px;
+            overflow: scroll;
+        }
+
+        @media screen and (max-width: 480px) {
+            td.overflow {
+                max-width: 0;
+                overflow: scroll;
+            }
+
+            .tableOverflow {
+                max-width: 480px;
+            }
+        }
+    </style>
+
+    <div class="tableOverflow">
+        <table class="center">
+            <tr>
+                <th colspan="3">Blink Codes</th>
+            </tr>
+            <tr>
+                <th>Animation (Click to animate)</th>
+                <th>LED State</th>
+                <th>Meaning</th>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='0' offtime='0' oncolor='black' offcolor='black'></div>
+                                          <div class='led' ontime='0' offtime='0' oncolor='black' offcolor='black'></div></div></td>
+                <td>LEDs Off</td>
+                <td>No Power</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='red' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='black' offcolor='red'></div></div></td>
+                <td>Blinking Alternating Red</td>
+                <td>Pigeon 2 does not have valid CAN.</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='black' offcolor='orange'></div></div></td>
+                <td>Blinking Alternating Orange</td>
+                <td>Pigeon 2 detects CAN and does not see robot enable. Phoenix is not running in robot controller <b>or</b> Pigeon 2 does not have good CAN connection to robot controller.</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div></div></td>
+                <td>Blinking Simultaneous Orange</td>
+                <td>Pigeon 2 detects CAN and does not see robot enable. Phoenix is running in robot controller <b>and</b> Pigeon 2 has good CAN connection to robot controller.</td>
```

Comment:
Same comments on before. I don't like using "robot enable" as I have no idea what that means... my robot hasn't been enabled?

Reply:
...yes, that is exactly what that means in this case. The robot is disabled, but the Pigeon 2 sees the robot controller and a robot program running Phoenix.